modules/awsparam.py

```diff
@@ -0,0 +1,85 @@
+#! /usr/bin/env python
+"""Salt module that retrieves a key stored in Amazon's Parameter store.
```

Comment:
Please standardize multi-line comments to use triple single quotes instead of triple double quotes.